os/src/FileOps.scala

```diff
@@ -280,8 +280,16 @@ object copy {
  * any files or folders in the target path, or
  * does nothing if there aren't any
  */
-object remove extends Function1[Path, Unit]{
-  def apply(target: Path): Unit = Files.delete(target.wrapped)
+object remove extends Function1[Path, Boolean]{
+  def apply(target: Path): Boolean = apply(target, false)
+  def apply(target: Path, checkExists: Boolean = false): Boolean = {
```

Comment:
Do we need 2 apply-es if we have a default value for `checkExists: Boolean = false`? 😅

Reply:
This one is needed for binary compatibility, and probably also for easier applicability, e.g. in `files.map(os.remove)`.

Reply:
Ah, ok, thx.